### PR TITLE
formdesign

### PR DESCRIPTION
**Pull request by mikehern**

### CODE REVIEW

**Comment by jmcelroy5** [comment]
@mikehern Nice!! Just one thing - we actually changed the file name to submission-form, so could you change these two files to be submission-form.css and submission-form.html then re-push? Sorry we didnt communicate that!

File: assets/css/new-project-form.css [hunk 1]
```diff
@@ -1,36 +1,58 @@
...
+}
+h2, h4{
+	color: #CBDEEF;
+}
+h5{
+	color: #a9a9a9;
+}
+p, div {
+    font-family: 'Montserrat', sans-serif;
+}
```
Comment (mikehern) [inline assets/css/new-project-form.css:13]: -Changed font and header theme

File: assets/css/new-project-form.css [hunk 2]
```diff
@@ -1,36 +1,58 @@
...
 .header {
 	padding: 2em;
-	box-shadow: 2px 2px 4px #D3D3D3;
-	border-radius: 0;
+	background-color: #020E32;
 }
 
 .publish {
-	width: 25vw;
-	border-radius: 0;
```
Comment (mikehern) [inline assets/css/new-project-form.css:9]: -Changed publish button

File: assets/css/new-project-form.css [hunk 3]
```diff
@@ -1,36 +1,58 @@
...
 .project {
 	padding: 1em;
 }
 
-.project span, .column h3, .upload-btn, .project-brief, .right-col-links a, .related-link, .autosave-text {
-	color: #10A1FF;
+.project span, .column h3, .upload-btn, .right-col-links a, .related-link, .autosave-text {
+	color: #17AC3B;
 	font-weight: 200;
-	text-shadow: 2px 2px 2px #e5e4e4;
```
Comment (mikehern) [inline assets/css/new-project-form.css:25]: -Stylized insert picture button

File: assets/css/new-project-form.css [hunk 4]
```diff
@@ -1,36 +1,58 @@
...
+	/*text-shadow: 2px 2px 2px #e5e4e4;*/
 }
 
 #property-image {
 	width: 100%;
 }
 
-.upload-btn, .project-brief-text, .community-benefits-text, .other-info-text, .supervisor-input {
-	border: 2px solid #10A1FF;
+.upload-btn{
```
Comment (mikehern) [inline assets/css/new-project-form.css:50]: -Customized form font

File: assets/css/new-project-form.css [hunk 5]
```diff
@@ -95,3 +119,8 @@
 .autosave-text {
 	margin-top: 18.7vh;
 }
+
+label {
```
Comment (mikehern) [inline assets/css/new-project-form.css:126]: -Stylized form labels

**Pull request closed without merging**